sass/components/_chart-controls.scss

```diff
@@ -212,3 +212,40 @@
         }
     }
 }
+@media only screen and (min-width: 480px) {
```

Comment:
> we could use CSS media queries to selectively hide the chart control labels, but turns out the queries only work for screen sizes. An alternative would be to append class names into the DOM when the chart is a certain width (see resizeScreen in ChartStore).

I've mentioned this in the card. We need to consider that the chart width does not necessary equal to the screen width. Consider binary-static (https://zakibinary.binary.sx/br_new-ui/en/app.html)

Reply:
@brucebinary you absolutely right , in this case I should add some class to `cq-context` so that I can recognize width of chart. what id your Idea about doing in this way ?

Reply:
@shayanbinary I'll leave it to you to decide. I think ChartIQ demo has their convention or smth (but we removed it already lol).